<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>690</width>
    <height>601</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Okno programu</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>-4</x>
      <y>-1</y>
      <width>701</width>
      <height>551</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Szyfrowanie</string>
     </attribute>
     <widget class="QProgressBar" name="progressBarCipher">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>480</y>
        <width>671</width>
        <height>31</height>
       </rect>
      </property>
      <property name="value">
       <number>24</number>
      </property>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>661</width>
        <height>101</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayoutCipher" columnstretch="0,1,0">
       <item row="1" column="0">
        <widget class="QLabel" name="label_5">
         <property name="font">
          <font>
           <family>Calibri</family>
           <pointsize>12</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Plik wyjściowy</string>
         </property>
         <property name="margin">
          <number>5</number>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="inputFileNameCipher">
         <property name="font">
          <font>
           <family>Consolas</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>&lt;plik&gt;</string>
         </property>
         <property name="margin">
          <number>5</number>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="outputFileNameCipher">
         <property name="font">
          <font>
           <family>Consolas</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>&lt;plik&gt;</string>
         </property>
         <property name="margin">
          <number>5</number>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="inputFileButtonCipher">
         <property name="text">
          <string>Wybierz plik...</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="outputFileButtonCipher_5">
         <property name="text">
          <string>Wybierz plik...</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <family>Calibri</family>
           <pointsize>12</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Plik wejściowy</string>
         </property>
         <property name="margin">
          <number>5</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>97</width>
        <height>144</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="font">
          <font>
           <family>Calibri</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Tryb Pracy</string>
         </property>
         <property name="margin">
          <number>5</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButtonECB">
         <property name="text">
          <string>ECB</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButtonCBC">
         <property name="text">
          <string>CBC</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButtonCFB">
         <property name="text">
          <string>CFB</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButtonOFB">
         <property name="text">
          <string>OFB</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>130</y>
        <width>132</width>
        <height>117</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="font">
          <font>
           <family>Calibri</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Długość klucza</string>
         </property>
         <property name="margin">
          <number>5</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton">
         <property name="text">
          <string>128 bit</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton_2">
         <property name="text">
          <string>192 bit</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton_3">
         <property name="text">
          <string>256 bit</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>130</y>
        <width>161</width>
        <height>65</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label_9">
         <property name="font">
          <font>
           <family>Calibri</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Długość podbloku</string>
         </property>
         <property name="margin">
          <number>5</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="subblockLength">
         <property name="readOnly">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>430</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>zaszyfruj</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>130</y>
        <width>258</width>
        <height>217</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="font">
          <font>
           <family>Calibri</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Wybierz odbiorcę:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="listWidgetUsers"/>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Deszyfrowanie</string>
     </attribute>
     <widget class="QProgressBar" name="progressBarDecipher">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>480</y>
        <width>671</width>
        <height>31</height>
       </rect>
      </property>
      <property name="value">
       <number>24</number>
      </property>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>661</width>
        <height>101</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayoutDecipher" columnstretch="0,1,0">
       <item row="0" column="0">
        <widget class="QLabel" name="label_7">
         <property name="font">
          <font>
           <family>Calibri</family>
           <pointsize>12</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Plik wejściowy</string>
         </property>
         <property name="margin">
          <number>5</number>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="inputFileNameDecipher">
         <property name="font">
          <font>
           <family>Consolas</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>&lt;plik&gt;</string>
         </property>
         <property name="margin">
          <number>5</number>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="outputFileNameDecipher">
         <property name="font">
          <font>
           <family>Consolas</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>&lt;plik&gt;</string>
         </property>
         <property name="margin">
          <number>5</number>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="inputFileButtonDecipher">
         <property name="text">
          <string>Wybierz plik...</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="outputFileButtonDecipher">
         <property name="text">
          <string>Wybierz plik...</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_8">
         <property name="font">
          <font>
           <family>Calibri</family>
           <pointsize>12</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Plik wyjściowy</string>
         </property>
         <property name="margin">
          <number>5</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>430</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>odszyfruj</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>258</width>
        <height>217</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <family>Calibri</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Wybierz odbiorcę:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="listWidgetUsers2"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>130</y>
        <width>146</width>
        <height>55</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QLabel" name="label_10">
         <property name="font">
          <font>
           <family>Calibri</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Wprowadź hasło:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditPassword">
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>690</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuZarz_dzanie">
    <property name="title">
     <string>Zarządzanie</string>
    </property>
    <addaction name="actionDodaj_uzytkownika_2"/>
   </widget>
   <widget class="QMenu" name="menuAplikacja">
    <property name="title">
     <string>Aplikacja</string>
    </property>
    <addaction name="actionO_aplikacji_2"/>
    <addaction name="separator"/>
    <addaction name="actionZako_cz_4"/>
   </widget>
   <addaction name="menuZarz_dzanie"/>
   <addaction name="menuAplikacja"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionDodaj_uzytkownika_2">
   <property name="text">
    <string>Dodaj użytkownika</string>
   </property>
  </action>
  <action name="actionO_aplikacji_2">
   <property name="text">
    <string>O aplikacji</string>
   </property>
  </action>
  <action name="actionZako_cz_4">
   <property name="text">
    <string>Zakończ</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
